style.css

```diff
@@ -131,14 +131,12 @@
 	fill: #c7c4c4;
 }
 
-.fileList_Virtual_Table_Row {
-	min-height: 27px;
-	height: auto !important;
-}
-
 .fileList_File_Name_Row {
-	padding-top: 6px;
-	padding-bottom: 6px;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+	direction: rtl;
```

Comment:
Do we specifically need this here? According to [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/direction)
> Note that text direction is usually defined within a document (e.g., with HTML's dir attribute) rather than through direct use of the direction property.

Reply:
If we want the ellipsis and truncation to be on the left side instead of the right side, yes. It's more of a CSS hack than anything. Documentation here: https://davidwalsh.name/css-ellipsis-left

Reply:
Oh weird; somehow I have even more questions about CSS 😭 . It is a cool little hack though!